src/webots/gui/WbStreamingServer.cpp

```diff
@@ -175,6 +178,15 @@ void WbStreamingServer::onNewTcpData() {
   }
 }
 
+void WbStreamingServer::sendTcpRequestReply(const QString &requestedUrl, QTcpSocket *socket) {
+  if (requestedUrl.startsWith("robot_windows/")) {
+    const QString fileName(WbProject::current()->path() + "plugins/" + requestedUrl);
```

Comment:
The availability of the robot windows for the web interface is specific to this branch, so nothing was working before as this branch introduced the feature.
I believe that the web interface is useful to develop specific windows (like competitions, benchmarks or specific applications) and I see no use case where we would want to publish a default PROTO robot window.
Moreover retrieving a default PROTO robot window at this point is very difficult because we don't know to which robot / PROTO it refers to.
Finally, introducing the [EXTERNPROTO feature](#1638) will totally change the way PROTO paths are managed.
So, I would postpone the implementation of this feature to whenever it's really needed.